### ISSUE TITLE
Windows Platform cannot use SH command

### ISSUE THREAD

**Issue by leejoo71** [Wikitude/wikitude-cordova-plugin-samples#49]
Windows Platform cannot use SH command in script directory

**Comment by AndreasSchacherbauerWikitude** [comment]
Hi @leejoo71,
Unfortunately you need to install our plugin on a macOS as the script requires some command line tools that are only available on macOS.

Best regards,
Andreas

**Comment by vicung** [comment]
I'm a bit confused because in your ReadMe.md there's a command for Windows and when I run it in my cloned repository I get a command "sh" not found. 
I've found this sample repository from the git Wikitude SDK repo https://github.com/Wikitude/wikitude-cordova-plugin but in the official cordova documentation on the Wikitude website there's this repo linked https://github.com/Wikitude/wikitude-sdk-samples although I can run the samples the AR isn't working that's why I wanted to try this repo.
Which one should I use when working on Windows with Cordova building for android platforms?

**Comment by AndreasSchacherbauerWikitude** [comment]
Hi @vicung,
https://github.com/Wikitude/wikitude-cordova-plugin contains our Cordova plugin
https://github.com/Wikitude/wikitude-cordova-plugin-samples contains our Cordova example app

https://github.com/Wikitude/wikitude-sdk-samples contains our Architect Worlds to which we link from time to time to explain our APIs in our examples and to customers

The Cordova example app contains the examples mentioned right before.

Regarding the script error on Windows: I'm looking into it...

**Comment by gaglianoalexis** [comment]
Hi,

Same problem here with the generator on Windows.
Do you have a Cordova Project ready?

Thank you,
Alexis Gagliano
Une Nouvelle Ville ASBL

**Comment by vicung** [comment]
@gaglianoalexis 
Yes, I have created a cordova project, added the Wikitude plugin and setup the samples manually, which works fine (except for the fact that the AR view / Live camera video stream is not working for me, which seems to be an other issue #52 ) 
I can still recommend downloading the samples and adding them manually to your cordova project though if you have trouble with the command line on Windows.
